web/js/search.js

```diff
@@ -0,0 +1,43 @@
+$(document).ready(function () {
+    $('#searchq').click(function(){
+        $("#divResult").fadeIn();
+    });
+
+    $(document).on("click", function(e) {
+        if (! $(e.target).hasClass("search")){
+            $("#divResult").fadeOut();
+        }
+    });
+});
+
+$(function(){
+    $("#searchq").keyup(function()
+    {
+        var searchTerm = $(this).val();
+        if (searchTerm != "")
+        {
+            var host = "http://localhost:888";
+            var url = host + "/api/search";
+            $.ajax({
+                type: "GET",
+                url: url,
+                data: "q=" + searchTerm,
+                cache: true,
+                success: searchSucces,
+                error: logError
+            });
+        }
+    });
+});
+
+var searchSucces = function(result){
+    $("#divResult").html(result).show();
+    $(".display_box").on("click", function(e){
+        var url = $(e.target).find("#referer").val();
+        window.location.href = "http://localhost:888" + url;
```

Comment:
Given you use the http://localhost:888 in two locations it's better to use a variable for ease of maintenance.